config/example/node.yaml

```diff
@@ -1,159 +1,159 @@
 logger:
-  level: debug
+  level: debug  # logger level: one of "info"(default), "debug", "warn", "error", "dpanic", "panic", "fatal"
 
 profiler:
-  address: 127.0.0.1:6060
-  shutdown_timeout: 15s
+  address: 127.0.0.1:6060  # endpoint for Node profiling
+  shutdown_timeout: 15s  # timeout for graceful profiling HTTP server shutdown
```

Comment:
graceful means it will not kick me out if I connect?

Reply:
it means that "graceful shutdown"(closing connection and so on) process has its own timeout before it is gonna be killed

I think I chosen wrong word order, made it as "# timeout for profiling HTTP server graceful shutdown"

Reply:
graceful shutdown timeout for HTTP profiling
but it still not clear, why it is "graceful"

Reply:
what exactly don't you like? term or its usage?

Reply:
What does "graceful" mean? 
For example nginx defines it as follow:
> Old worker processes close listen sockets and continue to service old clients. After all clients are serviced, old worker processes are shut down.

Is this true in our case?

Reply:
i used this term just because golang std lib use it:
```
// Shutdown gracefully shuts down the server without interrupting any
// active connections. Shutdown works by first closing all open
// listeners, then closing all idle connections, and then waiting
// indefinitely for connections to return to idle and then shut down.
// If the provided context expires before the shutdown is complete,
// Shutdown returns the context's error, otherwise it returns any
// error returned from closing the Server's underlying Listener(s).
```

this is exactly what `shutdown_timeout` is used for